packages/components/forms/src/Checkbox/README.mdx

```diff
@@ -71,76 +59,25 @@ We also provide the `Checkbox.Group` component that helps when using multiple ch
 - `value`: Same as `defaultValue` but this one sets the `isChecked` property, making the checkbox group controlled;
 - `onChange`: Handler that will be executed when any checkbox inside the group receives the event of change.
 
-```jsx
-<FormControl as="fieldset">
-  <FormControl.Label as="legend" marginBottom="none">
-    Checkbox group options
-  </FormControl.Label>
-  <Paragraph>Subtitle with more information if needed</Paragraph>
-  <Checkbox.Group name="checkbox-options" defaultValue={['option 1']}>
-    <Checkbox value="option 1" id="option-1" helpText="Some help text">
-      Option 1
-    </Checkbox>
-    <Checkbox value="option 2" id="option-2" helpText="Another help text">
-      Option 2
-    </Checkbox>
-  </Checkbox.Group>
-</FormControl>
-```
+```jsx file=../../examples/CheckboxGroupExample.tsx
```

Comment:
Could you please copy from the Radio documentation that it is a recommended way to deal with Checkbox component when they're used in forms.